locale/templates/LC_MESSAGES/django.pot

```diff
@@ -1,10 +1,14 @@
-
+# Translations template for addons-server.
+# Copyright (C) 2022 ORGANIZATION
+# This file is distributed under the same license as the addons-server project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
```

Comment:
should these have non-default values?

Reply:
I think we might want to change the `ORGANIZATION` bit but leave that line - it's supposed to be the "last translator" but it doesn't make sense to change in the template

(It's all autogenerated by `pybabel extract`)